pages/index.js

```diff
@@ -1,22 +1,45 @@
 import Head from "next/head";
+import { useState } from "react";
 import styled from "styled-components";
+import TripCard from "../components/TripCard";
+import { dbTrips } from "../db";
 
 export default function Home() {
+  const [trips, setTrips] = useState(dbTrips);
   return (
     <div>
       <Head>
-        <title>My App</title>
-        <meta name="description" content="Generated by create next app" />
-        <link rel="icon" href="/favicon.ico" />
+        <title>Trips</title>
       </Head>
-
+      <TripsHeadline>
+        Start
+        <br />
```

Comment:
You should not use br (mentioned by the coaches in between)

Reply:
As i understand we shouldn't use it for spacing elements. In this case it's really used for its purpose; a line break for text.